### PR TITLE
docs: landing-page added

### PR DESCRIPTION
**Pull request by rahulyadav139**
Demo: https://statuesque-creponne-891db8.netlify.app/

### CODE REVIEW

**Review by j24m — APPROVED** [review]
Code looks good to me. 
But from UI perpective your icons are a bit confusing , for example the camera icon is usually used for video calling , you could have instead used play button for videos page. You can maybe make changes later when you refactor. 
image

File: src/components/header/Header.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        <div className="hr-line fad"></div>
+        <ul className="hamburger-menu__items">
+          <li className="hamburger-menu__item">Home</li>
+          <li className="hamburger-menu__item">My Profile</li>
+          <li className="hamburger-menu__item">My Orders</li>
+        </ul>
+        <div className="btn-container">
+          <button className="btn primary">Logout</button>
+        </div>
+
```
Comment (manojsarna) [inline src/components/header/Header.js:29]: Suggestion : you could have divided the code into more components

File: src/components/header/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                <i className="fas fa-sun hidden"></i>
+              </span>
+            </label>
+          </div>
+        </li>
+
+        <li className="profile-item list-item">
+          <div className="avatar small">
+            <img
+              src="https://i.picsum.photos/id/704/536/354.jpg?hmac=k_PDx86tD-ILOtsUOKY9t5LAL5ycKiQ4ryVdlxhWoek"
```
Comment (manojsarna) [inline src/components/header/Navigation.js:26]: Suggestion : here also could have create more components

**Review by manojsarna — APPROVED** [review]
Overall code is good

UI Review : Its looking great 

Suggestion : Explore page needs spacing

File: src/components/header/Header.js (hunk 1 above)
Comment (rahulyadav139) [reply src/components/header/Header.js:29]: Sure, I'll do that. Thank you for the suggestion.

**Pull request merged**